src/protojure/grpc/client/utils.clj

```diff
@@ -31,7 +31,7 @@ after the channel is closed.  Used in remote procedure calls with unary paramete
 | **params**  | _any_                | The object to place on the channel                                         |
   "
   [ch params]
-  (-> (put ch params)
+  (-> (when params (put ch params))
```

Comment:
I think this will break if params == nil since there wont be a promise returned.  I think if you leave L34 alone, the nil params case is handled by your L19 change, while still returning a promise.